LLK/scratch-www
# fix: question mark at the end of faq header missing

### Resolves:

This resolves #6475

### Changes:

This adds a question mark at the end of `l10n.json` in Line 158 (`faq.createExtensionsTitle`). I'm not sure if this file is managed by a translation thing.

### Test Coverage:

1. Go to https://scratch.mit.edu/faq in your browser.
2. Notice that there is a question mark at the end of each heading
3. Scroll down to "Scratch Extensions"
4. Notice that "How do I create my own extension for Scratch" doesn't have a question mark at the end
